### PR TITLE
Add support for RPi/GPICase

### PR DESCRIPTION
**Pull request by mlnlbrt**
This adds a BOARD in RPi project to support Retroflag GPI CASE. 
It resolves https://github.com/libretro/Lakka-LibreELEC/issues/755

Thanks @duduke and @siuying for testing!

### CODE REVIEW

**Comment by natinusala** [comment]
Thanks for the PR! The GPICase is quite the hype right now.

I suppose that there is no way to merge it with the regular RPi project and enable/disable the case support with a switch at runtime?

**Comment by mlnlbrt** [comment]
@natinusala that is correct. We need a new BOARD for the GPICase, because stuff specific to it is configured in `distroconfig.txt`, which is loaded at boot.

**Comment by natinusala** [comment]
> By the way, it is still RPi project, I've just added a BOARD to it.

Sure, but that involves making a new build folder so a new image for it. We also need to update the website to add the GPICase RPi target.

It's not an issue, but I wanted to make sure we couldn't merge it with the existing project (without BOARD) to avoid doing all that.

**Comment by ToKe79** [comment]
Hi,
the build of this PR (897e9170994256c9fd508c7fbfc90fc3df94b9c4) finished successfully for all 45 targets.
Images for testing and logs are here: http://nightly.builds.lakka.tv/PRs/pr_763_897e917/

_This is an automated message_

**Comment by mlnlbrt** [comment]
@ToKe79 RPi.GPICase build has not been included here. Is something missing in my PR?

**Comment by ToKe79** [comment]
yup... I will run the job again and add BOARD=gpicase to the list of targets of the job. the list of targets is not dynamically expanded, it is just a list similar to the one in `pkg_all.sh`.

**Comment by natinusala** [comment]
@mlnlbrt can you PR fbalpha2012 separately for RPi0 and RPi1 please? I would like to have that in for the 2.3 release that's coming today / tomorrow

**Comment by ToKe79** [comment]
Hi,
this PR (5c8a02df50b539db1c01851e6b1241a8dcdd569f) did not build correctly for all targets.
Following targets failed (43):
- Generic.i386
- Generic.x86_64
- Generic_VK_nvidia.x86_64
- RPi2.arm
- RPi2.RPi4.arm
- Allwinner.Bananapi.arm
- Allwinner.Cubieboard2.arm
- Allwinner.Cubietruck.arm
- Allwinner.orangepi_2.arm
- Allwinner.orangepi_lite.arm
- Allwinner.orangepi_one.arm
- Allwinner.orangepi_pc.arm
- Allwinner.orangepi_plus.arm
- Allwinner.orangepi_plus2e.arm
- Allwinner.nanopi_m1_plus.arm
- imx6.cuboxi.arm
- imx6.udoo.arm
- OdroidC1.arm
- Odroid_C2.arm
- OdroidXU3.arm
- WeTek_Core.arm
- WeTek_Hub.arm
- WeTek_Play.arm
- WeTek_Play_2.arm
- S8X2.S82.arm
- S8X2.M8.arm
- S8X2.T8.arm
- S8X2.MXIII-1G.arm
- S8X2.MXIII-PLUS.arm
- S8X2.X8H-PLUS.arm
- S805.MXQ.arm
- S805.HD18Q.arm
- S805.M201C.arm
- S805.M201D.arm
- S805.MK808B-Plus.arm
- S905.arm
- S912.arm
- Rockchip.TinkerBoard.arm
- Rockchip.ROCK64.arm
- Rockchip.ROC-RK3328-CC.arm
- Rockchip.MiQi.arm
- Rockchip.ROCKPro64.arm
- Rockchip.ROCK960.arm

Logs and images for testing can be found here: http://nightly.builds.lakka.tv/PRs/pr_763_000069/

_This is an automated message_

**Comment by mlnlbrt** [comment]
@natinusala Reverted fbalpha2012 and mame2000 from this PR and added some final tweaks. It's ready to merge.

**Comment by natinusala** [comment]
I meant to have fbalpha2012 and mame2000 for the release, sorry for the confusion lol

If this PR is ready before the release then we'll have both in

**Comment by mlnlbrt** [comment]
@natinusala fbalpha2012 and mame2000 are back

File: packages/libretro/pcsx_rearmed/package.mk [hunk 1]
```diff
@@ -43,7 +43,7 @@ make_target() {
   if [[ "$TARGET_FPU" =~ "neon" ]]; then
     make -f Makefile.libretro HAVE_NEON=1 USE_DYNAREC=1 BUILTIN_GPU=neon
   elif [ "$ARCH" == "arm" ]; then
-    make -f Makefile.libretro HAVE_NEON=0 USE_DYNAREC=1
+    make -f Makefile.libretro HAVE_NEON=0 USE_DYNAREC=1 BUILTIN_GPU=unai
```
Comment (natinusala) [inline packages/libretro/pcsx_rearmed/package.mk]: Which platforms does that impact? Is that only RPi0 and RPi1? We don't want unai to be the renderer on all platforms

**Pull request merged**